Gemfile.lock

```diff
@@ -52,41 +52,41 @@ GEM
       bundler
       rake
       thor (>= 0.14.0)
-    arel (8.0.0)
+    arel (9.0.0)
     ast (2.4.0)
-    autoprefixer-rails (8.0.0)
+    autoprefixer-rails (8.4.1)
       execjs
     awesome_print (1.8.0)
     builder (3.2.3)
     bundler-audit (0.6.0)
       bundler (~> 1.2)
       thor (~> 0.18)
-    byebug (10.0.0)
-    capybara (2.18.0)
+    byebug (10.0.2)
+    capybara (3.1.0)
```

Comment:
Can we just update Capybara here? I'd like there to be just one commit fixing Capybara if that's what we need to be doing to fix CI.